tests/test_menu.py

```
@@ -8,28 +8,40 @@ def init_pygame(monkeypatch):
     """Initialize pygame for each test (headless-safe)"""
     monkeypatch.setenv("SDL_VIDEODRIVER", "dummy")
     monkeypatch.setenv("SDL_AUDIODRIVER", "dummy")
-    pygame.init()
-    pygame.font.init()
-    yield
-    pygame.quit()
+    """Tests for the simple Menu class behavior."""
 
+    import pytest
+    import pygame
```

Ruff (0.14.13)

Redefinition of unused `pygame` from line 2: `pygame` redefined here

Remove definition: `pygame`

(F811)